src/test/java/api/requests/RequestByInstanceIdResourceTests.java

```diff
@@ -168,6 +176,6 @@ private static JsonObject getLocationWithServicePoints(Set<UUID> servicePoints,
 
     location.put("id", UUID.randomUUID().toString());
 
-    return location;
+    return Location.from(location);
```

Comment:
Why are the API tests using the domain class?

Reply:
Replaced by JsonObject.

Reply:
Are you working on doing that for the other tests which also use the location domain class?

Reply:
Which tests do you mean? Seems like it was the only one place where the location domain class is used.

Reply:
There appears to still be multiple references to `Location` in `RequestByInstanceIdResourceTests.java`, maybe I have missed a further update

Reply:
`Location` is used in pair with `Item`. Here is an example:

`Item item1 = Item.from(ItemExamples.basedUponSmallAngryPlanet(bookMaterialTypeId, loanTypeId)
      .withTemporaryLocation(UUID.fromString(location1.getString("id")))
      .create())
      .withLocation(Location.from(location1))`

If we are going to get rid of it, the usage of domain `Item` class should be avoided as well. Looking at the code I see that it requires some efforts. I'm not sure that it is necessary to do it. What do you think?

Reply:
Yeah, this is my bad. This test isn't really an API test, and really shouldn't be here.

It definitely isn't the responsibility of this work to change that. Apologies :(